pilaf/package.json

```diff
@@ -12,7 +12,6 @@
     "build-storybook": "build-storybook"
   },
   "dependencies": {
-    "@dogehouse/kebab": "^0.2.0",
```

Comment:
gone

Reply:
i swear to god i added this

Reply:
`workspace:@dogehouse/kebab` is an invalid name so it gets removed, you probably meant to do
`"@dogehouse/kebab": "workspace:^0.2.4",`